packages/block-library/src/tag-cloud/style.scss

```diff
@@ -18,4 +18,12 @@
 		margin-left: 5px;
 		text-decoration: none;
 	}
+
+	&.is-style-pill a {
+		border: 1px solid var(--wp--preset--color--foreground);
```

Comment:
as much as I agree with this naming convention, we are assuming that foreground is going to be existing in the theme. I'd say at least give it a fallback when it's not the case?

Reply:
Can we go with just `currentColor` for border?

Reply:
^ Yeah, `currentColor` makes more sense here.

Reply:
Done